_locales/en/messages.json

```diff
@@ -481,6 +494,31 @@
     "message": "Remove section",
     "description": "Label for the button to remove a section"
   },
+  "setLinterLink": {
+    "message": "See a full list of rules",
+    "description": "Stylelint or CSSLint rules label added before a link"
+  },
+  "setLinterRulesTitle": {
+    "message": "Set rules for $linter$",
+    "description": "Stylelint or CSSLint popup header",
+    "placeholders": {
+      "linter": {
+        "content": "$1"
+      }
+    }
+  },
+  "resetLinterRules": {
```

Comment:
This is a totally generic label which we might reuse in the fure, rename it to "genereicResetLabel" and move it to the other generic labels.

Reply:
Done